components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/impl/APIStoreImpl.java

```diff
@@ -425,6 +430,29 @@ public Policy getPolicy(String policyLevel, String policyName) throws APIManagem
     }
 
     @Override
+    public Comment getCommentByUUID(String commentId, String apiId) throws APIManagementException {
+        Comment comment;
+        try {
+            comment = getCommentDAO().getCommentByUUID(commentId, apiId);
+        } catch (APIMgtDAOException e) {
+            String errorMsg = "Error occurred while retrieving label information";
```

Comment:
Add context information for the log message. You can add "apiId" and "commentId" to the log message.